tests/e2e/config/jest.config.js

```diff
@@ -45,15 +44,17 @@ if ( process.env.E2E_GROUP ) {
 		allowedPaths.push( e2ePaths[ process.env.E2E_GROUP ] );
 	}
 } else {
+	// The 'atomic' folder is a temporary measure until we are able to run all E2E tests for Atomic sites.
+	// The only way to run E2E tests locally for the Atomic site is to use E2E_GROUP=atomic, to avoid duplicate tests in other cases.
```

Comment:
Does this mean the default `npm run test:e2e` run certain tests twice?

Reply:
It used to ignore the `atomic` paths from the `allowedPaths` for the regular tests.
Not relevant anymore as the `atomic` folder is gone, the comment will be removed